keylime/measured_boot.py

```diff
@@ -67,17 +69,26 @@ def get_policy(mb_refstate_str):
 
 
 def evaluate_policy(
-    mb_policy, mb_policy_name, mb_refstate_data, mb_measurement_data, pcrsInQuote, pcrPrefix, agent_id
+    mb_policy: Optional[Policy],
```

Comment:
This `Optional` comes from `get_policy` above that may return None for mb_policy (line 66) ...